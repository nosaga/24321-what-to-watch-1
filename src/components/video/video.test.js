import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video.jsx";

const mock = {
  link: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  src: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  title: `Fantastic Beasts: The Crimes of Grindelwald`,
  genre: `family`
};

it(`Card renders correctly`, () => {
  const {
    link,
    src,
    title,
    genre
  } = mock;
  const tree = renderer
    .create(<VideoPlayer
      link={link}
      src={src}
      title={title}
      genre={genre}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
