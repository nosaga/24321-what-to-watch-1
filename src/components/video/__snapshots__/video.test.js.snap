// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<div>
  <button
    className="small-movie-card__play-btn"
    disabled={true}
    onClick={[Function]}
    type="button"
  >
    Play
  </button>
  <div
    className="small-movie-card__image"
  >
    <video
      alt="Fantastic Beasts: The Crimes of Grindelwald"
      height="175"
      muted={true}
      poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      width="280"
    >
      <source
        src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
        type="video/mp4"
      />
    </video>
  </div>
</div>
`;
